test/Chirp.Infrastructure.Tests/AuthorRepositoryTests.cs

```diff
@@ -91,6 +91,42 @@ public void CanCreateAuthorWhichExists(string name, string email)
         }
     }
 
+    [Theory]
+    [InlineData("Simon", "hejsameddejsa@gmail.com")]
+    [InlineData("Annabell", "f1skef1let@coldmail.com")]
+    [InlineData("Johnnie Calixto", "isbjørnogskruetrækker@hotmail.com")]
+    public void CanCreateAuthorWhereEmailExists(string name, string email)
+    {
+        //Arrange
+        AuthorDTO authorDTO = new AuthorDTO(name, email);
+
+        //Act and Assert
+        try
+        {
+            _repository.CreateAuthor(authorDTO);
+        }
+        catch (ArgumentException e)
+        {
+            Assert.Equal($"An author already exists with email: '{email}'", e.Message);
+        }
+    }
+
+    [Fact]
+    public void CanCreateAuthorWithLongName()
```

Comment:
Wondering if it should be changed to 'Cant' here as well to avoid confusion? (Might just be a matter of preference, though!)